src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,19 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid(
+    color: Color,
```

Comment:
Do you have color in Figure?

Reply:
The color field is in the Figure, we're using it here and adding specific parameters for the trapezoid in it's primery constructor

Reply:
so you can use private for this property